src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketRepository.java

```diff
@@ -77,7 +77,7 @@ public List<BitbucketNamedLink> getCloneUrls() {
 
     public Optional<BitbucketNamedLink> getCloneUrl(CloneProtocol cloneProtocol) {
         return cloneUrls.stream()
-                .filter(link -> Objects.equals(cloneProtocol.name, link.getName()))
+                .filter(link -> cloneProtocol.name.equalsIgnoreCase(link.getName()))
```

Comment:
can `cloneProtocol.name` ever be `null`? I believe that's why we originally had the cumbersome `Objects.equals` construct.

Reply:
Only if someone adds an enum in `CloneProtocol` that has a `null` name. Right now we only have http and ssh:
```java
public enum CloneProtocol {
    HTTP("http"),
    SSH("ssh");

    public final String name;

    private CloneProtocol(String name) {
        this.name = name;
    }
}
```